ivy/functional/backends/numpy/experimental/linear_algebra.py

```diff
@@ -149,3 +149,19 @@ def multi_dot(
 
 
 multi_dot.support_native_out = True
+
+
+def solve_triangular(
+    a: np.ndarray,
+    b: np.ndarray,
+    /,
+    *,
+    lower: bool = True,
+    adjoint: bool = False,
+    unit_diagonal: bool = False,
+    out: Optional[np.ndarray] = None,
+) -> np.ndarray:
+    return np.linalg.solve(a, b, out=out)
```

Comment:
You are doing a regular solve when instead you should take into account that the input is triangular.